packages/fxa-auth-server/lib/routes/password.js

```diff
@@ -588,6 +588,69 @@ module.exports = function (
         return {};
       },
     },
+    {
+      method: 'POST',
+      path: '/password/forgot/verify_otp',
+      options: {
+        ...PASSWORD_DOCS.PASSWORD_FORGOT_VERIFY_OTP_POST,
+        validate: {
+          payload: isA.object({
+            email: validators
+              .email()
+              .required()
+              .description(DESCRIPTION.emailRecovery),
+            code: isA
+              .string()
+              .length(config.passwordForgotOtp.digits)
+              .regex(validators.DIGITS),
+            metricsContext: METRICS_CONTEXT_SCHEMA,
+          }),
+        },
+      },
+      handler: async function (request) {
+        if (!config.passwordForgotOtp.enabled) {
+          throw error.featureNotEnabled();
+        }
+
+        log.begin('Password.forgotOtpVerify', request);
+        await request.emitMetricsEvent('password.forgot.verify_otp.start');
+        statsd.increment('otp.passwordForgot.attempt');
+
+        // TODO FXA-9487
+        // await customs.check(request, email, 'passwordForgotVerifyOtp');
+
+        request.validateMetricsContext();
+
+        const { email, code } = request.payload;
+        const account = await db.accountRecord(email);
+        const isValidCode = await otpManager.isValid(account.uid, code);
+
+        if (!isValidCode) {
+          // TODO FXA-9487
+          // increment failed attempt
```

Comment:
The customs check doesn't increment the count for this endpoint?

Reply:
hmm I think the "action" can be some arbitrary string; I'll need to double check.  But I do need to add some rate limiting/blocking config for this endpoint specifically.

Either way, I was overthinking it here with this particular comment.  The logic should just be attempts in time window; there's no need to explicitly track "failed" attempts with customs.